### ISSUE TITLE
Browser extension & native integration conflicting on Bitbucket Server

### ISSUE THREAD

**Issue by tsenart** [sourcegraph/sourcegraph#4900]
- **Sourcegraph version:** 3.6.0-rc.1

#### Steps to reproduce:

1. Point your Sourcegraph extension to https://k8s.sgdev.org
1. Go to https://bitbucket.sgdev.org/projects/SOUR/repos/vegeta/browse/encode.go?at=master#88
2. Hover over the `NewCSVEncoder` symbol and move the pointer horizontally over the symbol until two tooltips pop-up.

#### Expected behavior:

Only one tooltip shows up.

#### Actual behavior:

Two tooltips are shown, although with different content (note how the one in the back has the type of the argument as `io.Writer` and the one in the front only `Writer`).

![image](https://user-images.githubusercontent.com/67471/61296582-b2717580-a7da-11e9-9b8e-c041b0051a96.png)

**Comment by tsenart** [comment]
cc @keegancsmith

**Comment by lguychard** [comment]
I had tested for conflicts extensively with the dev extension, and hadn't found any issues, with the browser extension reliably detecting that the native integration was already running, and stopping execution:

![image](https://user-images.githubusercontent.com/1741180/61300525-688c8d80-a7e2-11e9-8971-bdee3998787e.png)

However, with the prod browser extension, conflict with the native integration can occur, with racy behaviour between content script injection & native integration init (see duplicate "detected code host bitbucket server" log):

![image](https://user-images.githubusercontent.com/1741180/61300881-1ac45500-a7e3-11e9-995e-57119f91e4fd.png)

**Comment by tsenart** [comment]
@nicksnyder handed over verification of this back to me. When should I take a look at it again?

**Comment by lguychard** [comment]
@tsenart I'm waiting for a final review from @felixfbecker on #4927. I'll ping you when it's merged & has made it into an RC.

**Comment by keegancsmith** [comment]
Yeah I am aware of this, once it lands I will cherry-pick and create a new RC. Our only "important" release blocker, so very keen to get it into the release :)

**Comment by christinaforney** [comment]
Reopening, since there is a new issue:
- There are two repo level sourcegraph buttons in BbS shown when extension and native extension are enabled
- Hovers are not conflicting, and only one button shown at the file level
- Command palette is only rendered once

![image](https://user-images.githubusercontent.com/1559622/61818930-ccc4d800-ae06-11e9-8cc8-91543a6c0b09.png)

**Comment by felixfbecker** [comment]
This may also be an issue with Gitlab, I've seen both logs being printed at the same time.

![image](https://user-images.githubusercontent.com/10532611/68679786-d2cef800-0560-11ea-8553-d549dbbfbd5f.png)
